sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -1,24 +1,44 @@
 -- Initialization query first observations for Firefox Android Clients.
-WITH first_seen AS (
+WITH baseline_clients AS (
   SELECT
     client_id,
     sample_id,
     first_seen_date,
     submission_date,
-    country AS first_reported_country,
+    country,
     isp AS first_reported_isp,
     DATETIME(first_run_date) AS first_run_datetime,
     normalized_channel AS channel,
     device_manufacturer,
     device_model,
     normalized_os_version AS os_version,
     app_display_version AS app_version,
-    locale
+    locale,
+    is_new_profile,
   FROM
-    `moz-fx-data-shared-prod.fenix.baseline_clients_first_seen`
+    `moz-fx-data-shared-prod.org_mozilla_firefox.baseline_clients_daily`
   WHERE
     submission_date >= '2019-01-01'
```

Comment:
`fenix.firefox_android_clients` has data for channel release from `2020-01-21`.

However I am noticing that `moz-fx-data-shared-prod.fenix.baseline_clients_first_seen` has data from `2019-06-21` for channel release, while `moz-fx-data-shared-prod.org_mozilla_firefox.baseline_clients_daily` has data from `2020-01-21`.

The older data comes from `moz-fx-data-shared-prod.org_mozilla_**fenix**.baseline_clients_first_seen` where the min. submission_date is `2019-06-21`.

I'm highlighting it to make sure we are happy to have data from this date or if we might need the older data in the future we'd need to join both sources. What do you think?